src/components/Card/CategoryCard/CategoryCard.scss

```diff
@@ -8,7 +8,6 @@
 }
 
 .iconWrapper {
-  // TODO (davidg): implement real icons
```

Comment:
This component isn't used anymore, it was in an earlier design. I'll delete the whole thing when the designs have stopped changing.